src/congpairs.cc

```diff
@@ -31,6 +31,7 @@ using libsemigroups::Congruence;
 using libsemigroups::word_t;
 using libsemigroups::relation_t;
 using libsemigroups::RecVec;
+using libsemigroups::Partition;
```

Comment:
It's used in the very next commit, in `CONG_PAIRS_NONTRIVIAL_CLASSES`.  Maybe I'll move this change to that commit to make this more readable.